frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -401,7 +402,20 @@ export const surveyLogic = kea<surveyLogicType>([
                                     AND properties.$survey_id = '${props.id}'
                                     AND timestamp >= '${startDate}'
                                     AND timestamp <= '${endDate}'
+                                    AND (
+                                        properties.$survey_completed = true
+                                        OR properties.$survey_submission_id == null
+                                    )
                                     ${answerFilter !== '' ? answerFilter : ''}
+                                    AND {filters}),
+                                    (SELECT COUNT(DISTINCT person_id)
+                                    FROM events
+                                    WHERE event = 'survey sent'
```

Comment:
can we extract those event names to a constant and stop duplicating them, same for similar hardcoded strings all over the place